### PR TITLE
chore: bump protocol buffers stubs

### PR DESCRIPTION
**Pull request by aldy505**
Closes # 

## ✅ Checks

- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -11,8 +11,8 @@ import (
 	"os/signal"
 	"time"
 
-	auth_jwt "kodiiing/auth/jwt"
-	auth_middleware "kodiiing/auth/middleware"
+	authjwt "kodiiing/auth/jwt"
+	authmiddleware "kodiiing/auth/middleware"
```
Comment (yogip07) [inline main.go:16]: ga pake camelCase?

**Review by yogip07 — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (aldy505) [reply main.go:16]: Goland nyuruh nya nyambung gitu

**Comment by yogip07** [comment]
ci failed

File: user/stub/user.go [hunk 2]
```diff
@@ -33,11 +33,12 @@ type Authentication struct {
...
-	JoinReasonWork       JoinReason = 1
-	JoinReasonFascinated JoinReason = 2
-	JoinReasonFriend     JoinReason = 3
-	JoinReasonOther      JoinReason = 4
+	JoinReasonJOIN_REASON_UNSPECIFIED JoinReason = 0
+	JoinReasonJOIN_REASON_SCHOOL      JoinReason = 1
+	JoinReasonJOIN_REASON_WORK        JoinReason = 2
+	JoinReasonJOIN_REASON_FASCINATED  JoinReason = 3
+	JoinReasonJOIN_REASON_FRIEND      JoinReason = 4
+	JoinReasonJOIN_REASON_OTHER       JoinReason = 5
```
Comment (WahidinAji) [inline user/stub/user.go]: kalau tujuan ngeletakin JoinReason di depan biar sesuai konteks, harusnya ga perlu. karena JOIN_REASON sudah mewakili. Masak mau kayak java -,-
```suggestion
	JOIN_REASON_UNSPECIFIED JoinReason = 0
	JOIN_REASON_SCHOOL      JoinReason = 1
	JOIN_REASON_WORK        JoinReason = 2
	JOIN_REASON_FASCINATED  JoinReason = 3
	JOIN_REASON_FRIEND      JoinReason = 4
	JOIN_REASON_OTHER       JoinReason = 5
```

Comment (aldy505) [reply user/stub/user.go]: Ini ubahnya jangan disini, ubah di repo proto2http.

Comment (yogip07) [reply user/stub/user.go]: jadinya diubah?

Comment (WahidinAji) [reply user/stub/user.go]: iya

Comment (aldy505) [reply user/stub/user.go]: ubahnya jangan disini woi

File: hack/stub/hack.go [hunk 3]
```diff
@@ -97,10 +97,11 @@ type Comment struct {
...
 const (
-	SortCriteriaScore        SortCriteria = 0
-	SortCriteriaComments     SortCriteria = 1
-	SortCriteriaCreated_date SortCriteria = 2
-	SortCriteriaUpdated_date SortCriteria = 3
+	SortCriteriaSORT_CRITERIA_UNSPECIFIED  SortCriteria = 0
+	SortCriteriaSORT_CRITERIA_SCORE        SortCriteria = 1
+	SortCriteriaSORT_CRITERIA_COMMENTS     SortCriteria = 2
+	SortCriteriaSORT_CRITERIA_CREATED_DATE SortCriteria = 3
+	SortCriteriaSORT_CRITERIA_UPDATED_DATE SortCriteria = 4
```
Comment (WahidinAji) [inline hack/stub/hack.go]: ini juga ;D

File: task/stub/task.go [hunk 4]
```diff
@@ -60,29 +33,111 @@ type StartTaskResponse struct {
...
+}
+
+type TaskDifficulty uint32
+
+const (
+	TaskDifficultyTASK_DIFFICULTY_UNSPECIFIED TaskDifficulty = 0
+	TaskDifficultyTASK_DIFFICULTY_EASY        TaskDifficulty = 1
+	TaskDifficultyTASK_DIFFICULTY_MEDIUM      TaskDifficulty = 2
+	TaskDifficultyTASK_DIFFICULTY_HARD        TaskDifficulty = 3
+)
```
Comment (WahidinAji) [inline task/stub/task.go]: ini juga

File: hack/stub/hack.go (hunk 3 above)
Comment (aldy505) [reply hack/stub/hack.go]: ubah di github.com/kodiiing/proto2http

File: task/stub/task.go (hunk 4 above)
Comment (aldy505) [reply task/stub/task.go]: ubah di github.com/kodiiing/proto2http

File: user/stub/user.go (hunk 2 above)
Comment (yogip07) [reply user/stub/user.go]: > ubahnya jangan disini woi

 iyaa wkwk

Comment (yogip07) [reply user/stub/user.go]: https://github.com/kodiiing/proto2http/pull/7

File: task/stub/task.go (hunk 4 above)
Comment (yogip07) [reply task/stub/task.go]: https://github.com/kodiiing/proto2http/pull/7

File: hack/stub/hack.go (hunk 3 above)
Comment (yogip07) [reply hack/stub/hack.go]: https://github.com/kodiiing/proto2http/pull/7

**Pull request merged**